gobblin-runtime/src/main/java/org/apache/gobblin/runtime/AbstractJobLauncher.java

```diff
@@ -221,17 +220,9 @@ public MetricContext apply(JobMetrics input) {
       JobExecutionEventSubmitter jobExecutionEventSubmitter = new JobExecutionEventSubmitter(this.eventSubmitter);
       this.mandatoryJobListeners.add(new JobExecutionEventSubmitterListener(jobExecutionEventSubmitter));
 
-      String eventMetadatadataGeneratorClassName =
+      this.multiEventMetadataGenerator = new MultiEventMetadataGenerator(
```

Comment:
Can we add a getStringList() method  in PropertiesUtils class (similar to ConfigUtils) and pass the list to MultiEventMetadataGenerator's constructor?